source/interface.cc

```diff
@@ -28,10 +29,53 @@ Interface<dim,spacedim,n_components,LAC>::
 set_time (const double &t) const
 {
   dirichlet_bcs.set_time(t);
+  // dirichlet_dot.set_time(t);
   forcing_terms.set_time(t);
   neumann_bcs.set_time(t);
+  // exact_solution.set_time(t);
```

Comment:
If they are not needed, remove the lines. Otherwise in a week from now, you'll see this and ask: why did I comment this code? was it needed? was it a bug that I solved?...